easybuild/framework/easyblock.py

```diff
@@ -802,8 +802,12 @@ def gen_builddir(self):
 
         builddir = os.path.join(os.path.abspath(build_path()), clean_name, self.version, lastdir)
 
+        if build_option('continue'):
+            self.log.debug("Setting cleanupoldbuild to False in continue mode")
+            self.cfg['cleanupoldbuild'] = False
+
         # make sure build dir is unique if cleanupoldbuild is False or not set
```

Comment:
update comment, also mention `or if --continue is used` (or just drop the comment, maybe...)